concept/answer/ConceptMap.java

```diff
@@ -48,14 +49,12 @@
 
 public class ConceptMap {
 
+    public static ConceptMap EMPTY = new ConceptMap(Collections.emptyMap());
```

Comment:
This might improve performance a bit - ConceptMaps are immutable and so the empty constructor can just be re-used as a singleton